ksqldb-functional-tests/src/test/resources/sql-tests/test.sql

```diff
@@ -0,0 +1,119 @@
+-- this test file is a "meta-test" that tests the basic functionality of the KsqlTester
+-- eventually, we plan to remove this test in favor of migrating the QTTs over to use this format
+-- directly
+
+----------------------------------------------------------------------------------------------------
+--@test: basic test
+----------------------------------------------------------------------------------------------------
+CREATE STREAM foo (id INT KEY, col1 INT) WITH (kafka_topic='foo', value_format='JSON');
+CREATE STREAM bar AS SELECT * FROM foo;
+
+INSERT INTO foo (rowtime, id, col1) VALUES (1, 1, 1);
+ASSERT VALUES bar (rowtime, id, col1) VALUES (1, 1, 1);
```

Comment:
Also, `Assert stream` and `Assert table` are missing

Reply:
yes, they are not implemented yet (see `AssertExecutor`)

Reply:
I just saw your comment in `AssertExecutor` so nvm